app/views/application/_signed_in_nav.html.haml

```diff
@@ -1,19 +1,26 @@
-%div.controls--contrast
-  %div.wrapper
+.controls--contrast
+  .wrapper
     %section.left
       %nav.controls
-        %p.inline-links--inverted
-          = link_to('Editorial', editorial_root_path)
-          - if @section
-            %b  /
-            = link_to @section.name, editorial_section_path(@section)
-        = yield :toolbar_actions
+        %header
+          = link_to(editorial_root_path) do
+            = image_tag('icons/editor.png')
+            %h1 Editor
+            %span Dashboard
+    
+        %ul
+          %li.dropdown
+            %a(href="#")
+              = image_tag('icons/human-small.png')
```

Comment:
Image_tag will default to using the filename as the alt-text. We could probably be more explicit with these icons if alt text isn't required.